tests/rptest/tests/rbac_test.py

```diff
@@ -632,7 +632,7 @@ def __init__(self, test_ctx, **kwargs):
         })
 
     def _has_license_nag(self):
-        return self.redpanda.search_log_any("Enterprise feature(s).*")
+        return self.redpanda.search_log_any("Enterprise features")
```

Comment:
nitpick: "Enterprise features" appears in an assert message [here](https://github.com/redpanda-data/redpanda/pull/23492/commits/9a5f7d7d58158ea38f3051a86544205004151f59#diff-6f212ba19edccd5c657400a0c2e9169b70a3dd81040388a732aaa7ebbe6566f3R64-R65). Could do

```suggestion
        return self.redpanda.search_log_any("Redpanda Enterprise features")
```

Reply:
Alternatively I could change the assert 😅

Reply:
Yeah, good point. I've changed the search text now to be more specific.